_includes/v20.1/backups/encrypted-backup-description.md

```diff
@@ -0,0 +1,11 @@
+<span class="version-tag">New in v20.1:</span> You can encrypt full or incremental backups by using the [`encryption_passphrase` option](backup.html#with-encryption-passphrase). Files written by the backup (including `BACKUP` manifests and data files) are encrypted using the specified passphrase to derive a key. To restore the encrypted backup, the same `encryption_passphrase` option (with the same passphrase) must included in the [`RESTORE`](restore.html) statement.
+
+When used with [incremental backups](backup.html#incremental-backups), the `encryption_passphrase` option is applied to all the [backup file URLs](backup.html#backup-file-urls), which means the same passphrase must be used when appending another incremental backup to an existing backup. Similarly, when used with [locality-aware backups](backup.html#create-locality-aware-backups), the passphrase provided is applied to files in all localities.
+
+Encryption is done using [AES-256-GCM](https://en.wikipedia.org/wiki/Galois/Counter_Mode), and GCM is used to both encrypt and authenticate the files. A random per-file [salt](https://en.wikipedia.org/wiki/Salt_(cryptography)) is used to derive per-file [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) keys from the specified passphrase.
```

Comment:
We actually changed this a bit after the initial draft in a really minor way: we now use a random salt to derive a key _once per backup_, and then use a random _IV_ (initialization vector) per-file. It's a small change, but the people who actually read this will probably care. They might also want to know that we use pbkdf2 with 64000 iterations for the key derivation.